atd-vze/src/_nav.js

```diff
@@ -21,15 +21,10 @@ export default {
       icon: "icon-shield",
       children: [
         {
-          name: "Crashes",
+          name: "All Crashes",
```

Comment:
Can we go one step further and remove these children all together and just make line 20 above link to the `/crashes` url?